docs/src/APIs/BalanceLaws/BalanceLaws.md

```diff
@@ -10,15 +10,23 @@ CurrentModule = ClimateMachine.BalanceLaws
 BalanceLaw
 ```
 
+## State variable types
+
+```@docs
+Conservative
```

Comment:
I've mentioned this before, but what we call the "conservative state" isn't really conservative. Its really just the state or field or something like that.

Not sure what a better name would be.

- `Field`
- `State`
- ????

Reply:
Isn't this for the conserved variables, though? Maybe `Prognostic`?

Reply:
We have sources and sinks, so it's not conserved.

Reply:
I think conservative means different things in applied math versus GFD?

I'm fine with `Prognostic` instead. I mainly needed something to distinguish it from the other state vectors we have so chose conservative

Reply:
I see. Hmm... we're aiming for these variables to be conserved. I'd like to call it `Conserved`, but I agree that it's not strictly conserved (since `source!` is now potentially a function of gradients) so it's probably better to avoid the misnomer. I think `Prognostic` is a good name, I'll change it.